src/chapter3/challenges.md

```diff
@@ -1 +1,13 @@
 # Challenges
+
+## Challenge 1
+
+Something simple to start off. Create a bash script called `hello.sh` that prints "Hello World" to the screen. Submit this job to the queue using `sbatch`. Check the status of the job using `squeue`. Once the job has finished, check the output using `cat`. You can find the output file in the directory you submitted the job from.
```

Comment:
Can you edit this challenge to include cloning the challenge repo onto m3, and have them create the slurm script in the relevant directory run the job and commit there work etc.